readme.md

```diff
@@ -124,7 +124,7 @@ $publisher->updateEdition(6894180, [
     "onDeviceName" => "My Edition",
     "shortName" => "My Edition",
     "drmEnabled" => "true",
-    "iosSaleEnabled" => "true",
+    "iosSaleEnabled" => "FREE",
```

Comment:
I think this option might still be wrong? According to the Publisher docs, this fields is called 'iosSaleOption'.